installer/scripts/omsadmin.sh

```diff
@@ -239,7 +242,7 @@ parse_args()
 {
     local OPTIND opt
 
-    while getopts "h?s:w:d:vp:u:a:lx:XUm:Nrn:oR" opt; do
+    while getopts "h?s:w:d:vp:u:a:lx:XUm:Nrn:oRc" opt; do
```

Comment:
nit: it's a very small thing, but what I had previously thought was that the "c" should be included with N, r, and n which also pertain to the process limit; like so: `...:Nrcn:...` . The "c" char cannot be right before a colon, because that indicates that it needs to have a following parameter (like n does for setting the limit).

Reply:
That's why I like alphabetical.  Much of reference needs are primarily around finding things, rather than buttressing meaning.  However, in this case yes makes good sense given how we do it.  I'd say since reporting is often most upfront, I'd say cNrn might be fine.  I'll do that, but tell me if you can think of a better one.

Reply:
I don't know why I didn't see your EXACT sequence the first time.  If there are any more changes, I'll make it like that, but for now I have a working build.